tests/data_context/store/test_datasource_store_cloud_backend.py

```diff
@@ -90,6 +98,61 @@ def json(self):
         )
 
 
+@pytest.mark.cloud
+@pytest.mark.unit
+def test_datasource_store_get_by_name(
+    ge_cloud_base_url: str,
+    ge_cloud_organization_id: str,
+    request_headers: dict,
+    datasource_config: DatasourceConfig,
+    datasource_store_ge_cloud_backend: DatasourceStore,
+) -> None:
+    """What does this test and why?
+
+    The datasource store when used with a cloud backend should emit the correct request when getting a datasource with a name.
+    """
+
+    id_: str = "example_id_normally_uuid"
+    datasource_name: str = "example_datasource_config_name"
+
+    def mocked_response(*args, **kwargs):
+        class MockResponse:
+            def __init__(self, json_data: dict, status_code: int) -> None:
+                self._json_data = json_data
+                self._status_code = status_code
+
+            def json(self):
+                return self._json_data
+
+        return MockResponse(
+            {
+                "data": {
+                    "id_": id_,
+                    "attributes": {"datasource_config": datasource_config},
+                }
+            },
+            200,
+        )
```

Comment:
I moved all of this to a `conftest` and used a `pytest.fixture`. Let's hash out what the best course of action is here.